docs/documentations/librería-de-aserción.md

```diff
@@ -0,0 +1,8 @@
+# Librería de aserción
+
+Para la librería de aserción, el criterio de elección se centra principalmente en la frescura con la intención de utilizar una librería que esté más actualizada.
+- [**should.js**](https://github.com/shouldjs/should.js):  librería de aserción de estilo BDD, actualmente no se encuentra en mantenimiento pues el último commit se realizó en 2019. 
+- [**Chai**](https://www.chaijs.com/): librería de aserción BDD/TDD popular y extendida, proporciona un sintaxis natural  que ayuda a la generación de los tests. Se encuentra actualizada con el último commit realizado hace tan sólo 12 días.
+- [**Unexpected**](https://unexpected.js.org): librería de aserción BDD que también utiliza la sintaxis _expect_, es menos conocida que las dos opciones anteriores pero se encuentra en mantenimiento con el último commit realizado en el octubre de este año.
+
+Dada las opciones anteriores, optamos por la elección de **Chai** por ser la más actualizada.
```

Comment:
¿Ninguna más? ¿Seguro?

Reply:
Están también expect.js pero le ocurre lo mismo que con should.js.

Reply:
Si le ocurre lo mismo, o las borras las dos o las incluyes las dos.

Reply:
He añadido ya expect.js en la documentación.